File: basic arithmetic/armstrong.py
def checkArmstrong(n):
    sum=0
    dummy=n
    while(n>0):
        ld=n%10
        sum=sum+(ld*ld*ld)
        n=n//10
    if(sum==dummy):
        return True
    else:
        return False
n = int(input()) 
if checkArmstrong(n):
    print("true")
else:
    print("false")

    

File: basic arithmetic/armstrong.cpp
#include <iostream>
using namespace std;

bool checkArmstrong(int n){
	//Write your code here
	int ld,sum=0;
	int org=n;
	while(n>0){
		ld=n%10;
		sum=sum+(ld*ld*ld);
		n=n/10;
	}
	return (org==sum);
}

int main(){
	int n;
	cin>>n;
	if(checkArmstrong(n)){
		cout<<"true";
	}
	else{
		cout<<"false";
	}
	return 0;
}
